python/BUILD

```diff
@@ -14,3 +14,16 @@
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
+
+# Python2 is a dependency of rules_docker due to its dependency on
+# containerregistry. To run builds on environments in which 'which python'
+# points to python3, you can use the py_runtime target below.
+# To use this target, simply add the following flag to your build commands:
+# --python_top=//python:py_runtime_2
```

Comment:
What if you are building a python3 binary and packaging in a docker image in the same bazel command?

Reply:
Unfortunately, Bazel does not quite support running builds with python2 and python3 at the same time (as far as I know). This is one of the outstanding problems for python support with Bazel that has been around for quite some time. I recommend you open an issue to ask about support for this use case (these are some of the old issues I saw related to this topic: https://github.com/bazelbuild/bazel/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+python2+python3)

Reply:
Which actually is the main problem. For repos with python3 code, they have already set their python_path flag to point to python3 which makes building python2 code impossible. If it's just a question of "python" on people's machine pointing to python3, but all their monorepo code is python2, then maybe they are already setting the python_path or python_top.